build_scripts/build_all_courses.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if test -f ".env"; then
+  export $(cat .env | xargs)
+fi
+
+if [[ -z $OCW_TO_HUGO_OUTPUT_DIR ]]; then
+  echo "OCW_TO_HUGO_OUTPUT_DIR not set"
+  exit 1
+elif [[ -z $COURSE_OUTPUT_DIR ]]; then
+  echo "COURSE_OUTPUT_DIR not set"
+  exit 1
+elif [[ -z $COURSE_BASE_URL ]]; then
+  echo "COURSE_BASE_URL not set"
+  exit 1
```

Comment:
probably good to add a check for `VERBOSE` here too

Reply:
I did this and also moved the unbound check the beginning, defaulting it to zero for the rest of the file if it's not set.  For what it's worth, the flag is mostly there to prevent Hugo from spitting out statistics about every course build.